src/themes/masschallenge/elements/container.overrides

```diff
@@ -0,0 +1,24 @@
+/*******************************
+         Theme Overrides
+*******************************/
+
+@media screen and (min-width: 768px) and (max-width: 1440px) {
+.container{
```

Comment:
Our selectors can be more specific for consistency with Semantic UI i.e. `.ui.container`

Reply:
When I use the `.ui.container` it still references the Semantic UI default values that is why I had used container instead.

Reply:
I have made the changes as suggested